coding-concepts.js

```diff
@@ -18,38 +18,38 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: using the .length function counts the characters including the blank space
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: o 
+// b) Verify and explain: this is searching for the 4th character starting with 0
 
 // --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
+//const languages = ["JavaScript", "Ruby", "Python", "C++"]
+//const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: The out put will be Ruby as it's pulling from the Array. 0,1,2,3,
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: SATURDAY, SUNDAY
+// b) Verify and explain: .toUpperCase will capitalize everything in the string.
```

Comment:
This will return a TypeError. This is because `.toUpperCase()` only works on strings. In the code, it is being applied to an array, which will cause an error.